blog/2024-12-10-pydantic.md

```diff
@@ -0,0 +1,14 @@
+---
+author: Joost Lekkerkerker
+authorURL: https://github.com/joostlek
+authorImageURL: https://avatars.githubusercontent.com/u/7083755?v=4
+title: "Pydantic bump to v2"
```

Comment:
```suggestion
title: "Pydantic v2 mandatory from Home Assistant Core 2025.1"
```
Maybe something like this?

Reply:
`mandatory` sounds like everything should use it, which is incorrect, it's only for pydantic v1-based intergation

Reply:
But I do agree the current one is subpar, so open for other suggestions. We should both be trying to be a reason for action, but also make sure we target the right people

Reply:
Indeed, poor choice of words.. How about  "Dropping support for pydantic v1 in 2025.1"?

Reply:
"Moving from Pydantic v1 to v2"? (in 2025.1)?

Reply:
Sounds also good, I'd keep the version somehow in the title as it's a clear indication when the change happens.